lib/Conf.ml

```diff
@@ -1371,6 +1371,10 @@ let output =
       & opt (some string) default
       & info ["o"; "output"] ~doc ~docs ~docv)
 
+let partial =
+  let doc = "Ignore invalid (unparsable) parts of the input." in
+  mk ~default:false Arg.(value & flag & info ["partial"] ~doc ~docs)
```

Comment:
I'm not too sure about the name. `--partial` can convey the idea that only part of the file will be kept, or that formatting will be done only on one part of the file. I think that the idea of the configuration key is to enable this particular feature, so what do you think about (combinations of) the following options:
- `--support-invalid-files`
- `--recover-parse-errors`
- `--format-invalid-code`

By the way, it seems that historically all flags (boolean options) tend to evolve into something non-boolean. For this particular feature, I could imagine that we expose different granularities of recovery when we try to introduce recovery at the expression level later... but not sure.

Should we account for this and start with a `<never> | <auto>` enum?

Reply:
Agreed, I renamed to `--format-invalid-files={never,auto}`